### ISSUE TITLE
IndentationCheck false positive conflict - reports 2 opposite failures

### ISSUE THREAD

**Issue by gratiartis** [checkstyle/checkstyle#6166]
Correcting the indentation level of an array of values being passed as arguments to a method, results in an opposite failure.

Given this config using checkstyle 8.14:
```
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC "-//Puppy Crawl//DTD Check Configuration 1.2//EN"
        "http://www.puppycrawl.com/dtds/configuration_1_2.dtd">
<module name="Checker">
	<module name="TreeWalker">
		<module name="Indentation">
			<property name="forceStrictCondition" value="true" />
		</module>
	</module>
</module>
``` 
... and this class:
``` 
package com.myproject.config;
import org.springframework.web.servlet.config.annotation.CorsRegistry;
public class CorsConfigMin {
    public void addCorsMappings(CorsRegistry reg) {
        reg.addMapping("/**")
            .allowedHeaders("Origin", "x-requested-with", "Content-Type", "Authorization", "Accept",
            "Access-Control-Request-Method", "Access-Control-Request-Headers");
    }
}
``` 
Reports incorrect indentation:
``` 
<error line="10" severity="error" message="&apos;method call&apos; child has incorrect indentation level 12, expected level should be 16." source="com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck"/>
``` 
BUT ... changing the indentation to 16 characters reports the opposite:
```
<error line="9" severity="error" message="&apos;&quot;Access-Control-Request-Method&quot;&apos; has incorrect indentation level 16, expected level should be 12." source="com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck"/>
```
It should be possible to correct the indentation failure to the expected level and have the code pass the check. I have reproduced this error in 8.14 and 6.18 (default with Maven plugin).

Java version: 1.8.0_151, vendor: Oracle Corporation

**Comment by romani** [comment]
@gratiartis , please use our CLI and report issue as demanded in issue template, line 9 and 10 are not in sync with code you provided.
